public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js

```diff
@@ -475,7 +475,15 @@ function Onboarding(svl, actionStack, audioEffect, compass, form, handAnimation,
         var mission = missions[0];
 
         missionContainer.setCurrentMission(mission);
-        modalMission.setMissionMessage(mission, neighborhood);
+        if (missionContainer.isTheFirstMission()) {
+            svl.initialMissionInstruction = new InitialMissionInstruction(svl.compass, svl.map,
+                svl.neighborhoodContainer, svl.popUpMessage, svl.taskContainer, svl.labelContainer);
+            modalMission.setMissionMessage(mission, neighborhood, null, function () {
+                svl.initialMissionInstruction.start(neighborhood);
+            });
+        }else{
+            modalMission.setMissionMessage(mission, neighborhood);
+        }
```

Comment:
Try this code with registered users, it shows the mission box twice. The way to recreate it - log in and load audit page. Before clicking the OK button of the mission dialog box, click Retake Tutorial. At the end of the tutorial, the mission dialog is shown once from Onboarding.js and next from Main.js.